Directory.Build.props

```diff
@@ -17,6 +17,10 @@
     <DisableImplicitPackageTargetFallback>true</DisableImplicitPackageTargetFallback>
 
     <!-- <ArtifactsShippingSymbolsDir>$(ArtifactsDir)symbols\$(Configuration)\Shipping</ArtifactsShippingSymbolsDir> -->
+
+    <!-- Flag to enable using Serilog for debug logging for how SDK resolvers are called -->
+    <EnableSerilogLogging>false</EnableSerilogLogging>
```

Comment:
The only usage is during development? You build with this flag enabled and then copy paste to VS, run it manually and see the result?